Test/git-issues/github-issue-3343.dfy

```diff
@@ -0,0 +1,6 @@
+// RUN: %baredafny run "%s" -t:java > "%t"
```

Comment:
Consider using `%testDafnyForEachCompiler` instead - should be the default for any compilable test case, and especially good for this one since it showed up at compiler time for each compiler.